mmv1/provider/terraform_validator.rb

```diff
@@ -282,7 +282,9 @@ def copy_common_files(output_folder, generate_code, _generate_docs)
                        ['converters/google/resources/iam_spanner_instance.go',
                         'third_party/terraform/utils/iam_spanner_instance.go'],
                        ['converters/google/resources/spanner_instance_iam.go',
-                        'third_party/validator/spanner_instance_iam.go']
+                        'third_party/validator/spanner_instance_iam.go'],
+                        ['test/utils.go',
```

Comment:
```
Offenses:

provider/terraform_validator.rb:286:25: C: Layout/ArrayAlignment: Align the elements of an array literal if they span more than one line.
                        ['test/utils.go', ...
                        ^^^^^^^^^^^^^^^^^
provider/terraform_validator.rb:287:25: C: Layout/ArrayAlignment: Align the elements of an array literal if they span more than one line.
                        'third_party/validator/tests/utils.go']
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

```suggestion
                       ['test/utils.go',
```

I think this will fix it?